workflow-templates/test-package.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Simple Package Testing Workflow (dart-lang)",
```

Comment:
We usually don't use title casing, but sentence casing in Dart communication, so 'Simple package testing workflow'

Reply:
Done.